tests/main/views/test_services.py

```diff
@@ -273,7 +273,7 @@ def test_expired_framework_causes_service_to_have_unavailable_banner(self):
         assert unavailable_banner.body_text() == 'The {} framework expired on {}. Any existing contracts with {} are' \
             ' still valid.'.format(
                 self.service['services']['frameworkName'],
-                'Tuesday 5 January 2016',
+                'Thursday 6 January 2000',
```

Comment:
I'm not 100% sure where this test date is coming from - where are we setting `frameworkExpiresAtUTC`?

Reply:
@katstevens Good question! It was coming from this fixture: https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/be59e960522a02ff26e03e0a597400ca35d9c3b3/tests/main/views/test_services.py#L40 
I have updated the test now in be59e96 to update this value explicitly so it is clearer what value is being read.

Reply:
Aha - from the `FrameworkStub`, gotcha. Good call on setting an explicit value 👍